Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2005-06-08
Initial Package Version: 2.12q
Origin: Jim Gifford
Upstream Status: Sent
Description: Fixes compile issue under MIPS 64 bit
 
diff -Naur util-linux-2.12q.orig/fdisk/sfdisk.c util-linux-2.12q/fdisk/sfdisk.c
--- util-linux-2.12q.orig/fdisk/sfdisk.c	2005-01-04 22:31:57.000000000 +0000
+++ util-linux-2.12q/fdisk/sfdisk.c	2005-06-08 23:15:17.000000000 +0000
@@ -172,7 +172,7 @@
  * Note: we use 512-byte sectors here, irrespective of the hardware ss.
  */
 #undef use_lseek
-#if defined (__alpha__) || defined (__ia64__) || defined (__x86_64__) || defined (__s390x__)
+#if defined (__alpha__) || defined (__ia64__) || defined (__x86_64__) || defined (__s390x__) || defined (__mips__)
 #define use_lseek
 #endif
 
